sx-search.el

```diff
@@ -61,6 +62,16 @@ KEYWORDS is passed to `sx-method-call'."
     :auth t
     :filter sx-browse-filter))
 
+(defconst sx-search--order-methods
+  (cons '("Relevance" . relevance)
+        (cl-remove-if (lambda (x) (eq (cdr x) 'hot))
+                      (default-value 'sx-question-list--order-methods)))
+  "Alist of possible values to be passed to the `sort' keyword.")
+
+(defvar sx-search-default-order 'activity 
```

Comment:
Should this eventually be a `defcustom`? I don't know how Customize would handle variadic options (perhaps some validation function?), but this seems like a variable the user would be interested in setting.